lib/ios/reactnativeuilib/dynamicfont/DynamicFont.m

```diff
@@ -0,0 +1,114 @@
+#import "DynamicFont.h"
+#import <CoreText/CoreText.h>
+#import <UIKit/UIKit.h>
+#import <Foundation/Foundation.h>
+
+@implementation DynamicFont
+
+- (dispatch_queue_t)methodQueue
+{
+    return dispatch_get_main_queue();
+}
+RCT_EXPORT_MODULE()
+
+RCT_EXPORT_METHOD(
+  loadFontFromFile:(NSDictionary *)options callback:(RCTResponseSenderBlock)callback)
+{
+    NSString *path = [options valueForKey:@"path"];
```

Comment:
Here - the parm name you extract is "path" (not "pathFile")

Reply:
I'll comment it out